backend/UnicornRewards.API/Controllers/ContactController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using UnicornRewards.API.Services;
+
+namespace UnicornRewards.API.Controllers
+{
+    [ApiController, Route("api/contact"), AllowAnonymous]
```

Comment:
Why 'AllowAnonymous' if the auth configuration is in place?

Reply:
AllowAnonymous removed, change updated on "fixes to contact list functionality" commit